### DISCUSSION TITLE
Move AMT only computer to Agent Based Group with Fully Automatic AMT enabled settings

### DISCUSSION THREAD

**Discussion by shinomen** [Ylianst/MeshCentral#6302] [Q&A]
Hello,

I originally only planned ot use MeshCentral for AMT only management so that's how I setup my groups and joined the computers to mesh.   Now that I have learned how the Mesh Agent can fully provision devices' AMT settings, I am creating Agent Groups and joining computers to that using the mesh agent installer.

However, I have existing computers that I want to move over to the new Agent groups from the agentless groups.  Simply moving the device did not work.  I selected the device in the agent only group and did the move operation to the agent group but it did not move.  I assumed that because it didn't have an agent installed already it wasn't going to move.  So I installed an agent for a new group on that AMT only computer and it shows up under the new group but shows Agent for it's connection type.  The AMT device still shows under the old AMT only group.  I attempted to remove the computer from the group thinking that it would re-provision and join the correct group based on the agent being installed.  However, it just reported back and showed up under the old AMT only group with CIRA as the connection type and still shows as Agent connection type under the Agent group.

So my question is how do I get the device to show only under the Agent group with both Agent connection and CIRA connection?

**Comment by si458** [comment]
Yes sadly you can't move amt devices to different groups! You will have to unprovision/deactivate the amt for the group/devices, let them disconnect. Delete the devices from the amt group. Then install meshagent on the devices and reactivate amt in the meshagent groups
